Traceback (most recent call last):
  File "/projectnb/ds563/hsharma/DeepCore/main.py", line 319, in <module>
    main()
  File "/projectnb/ds563/hsharma/DeepCore/main.py", line 270, in main
    prec1 = test(test_loader, network, criterion, epoch, args, rec)
  File "/projectnb/ds563/hsharma/DeepCore/utils.py", line 82, in test
    for i, (input, target) in enumerate(test_loader):
  File "/projectnb/ds563/hsharma/ml_env/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 359, in __iter__
    return self._get_iterator()
  File "/projectnb/ds563/hsharma/ml_env/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 305, in _get_iterator
    return _MultiProcessingDataLoaderIter(self)
  File "/projectnb/ds563/hsharma/ml_env/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 918, in __init__
    w.start()
  File "/projectnb/ds563/hsharma/ml_env/lib/python3.9/multiprocessing/process.py", line 121, in start
    self._popen = self._Popen(self)
  File "/projectnb/ds563/hsharma/ml_env/lib/python3.9/multiprocessing/context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "/projectnb/ds563/hsharma/ml_env/lib/python3.9/multiprocessing/context.py", line 277, in _Popen
    return Popen(process_obj)
  File "/projectnb/ds563/hsharma/ml_env/lib/python3.9/multiprocessing/popen_fork.py", line 19, in __init__
    self._launch(process_obj)
  File "/projectnb/ds563/hsharma/ml_env/lib/python3.9/multiprocessing/popen_fork.py", line 66, in _launch
    self.pid = os.fork()
OSError: [Errno 12] Cannot allocate memory
